x/gastracker/params.go

```diff
@@ -25,6 +28,9 @@ type Params struct {
 	GasRebateSwitch               bool
 	GasRebateToUserSwitch         bool
 	ContractPremiumSwitch         bool
+
+	InflationRewardQuotaPercentage uint64
+	GasRebatePercentage            uint64
```

Comment:
both these params should be sdk.Dec

and this should be defined as a protobuf.

Reply:
Why would we want them to be sdk.Dec?

Percentage range would be between 0 to 100 integer.

Reply:
it's a ratio, going from 0 to 1

Reply:
In the current implementation in master, it is a int from 0 to 100 and taken as percentage value. If we change it to ratio the only advantage I see is we can have more resolution in percentage (50.34%), but otherwise it is same thing.